common/script/script-api/src/main/java/org/thingsboard/script/api/tbel/TbDate.java

```diff
@@ -167,8 +488,9 @@ public static long parse(String value, String format) {
             return -1;
         }
     }
-
     public static long parse(String value) {
+        DateTimeFormatter isoDateFormatter = DateTimeFormatter.ofPattern(
```

Comment:
Move to a static variable (constant) to avoid creation of the formatter on every method execution